src/components/Calender.vue

```diff
@@ -185,6 +208,7 @@
 		{
 			font-size: 1.3rem;
 			transform: translate(-3px, -3px);
+			margin-bottom: auto;
```

Comment:
Ah I see you do add it back in, but a little too late. These styles only kick in after 975px, while the titles disappear at 1400.